src/Microsoft.ML.AutoML/Experiment/Experiment.cs

```diff
@@ -112,19 +112,8 @@ private static DirectoryInfo GetModelDirectory(DirectoryInfo rootDir)
             {
                 return null;
             }
-            var subdirs = rootDir.Exists ?
-                new HashSet<string>(rootDir.EnumerateDirectories().Select(d => d.Name)) :
-                new HashSet<string>();
-            string experimentDir;
-            for (var i = 0; ; i++)
-            {
-                experimentDir = $"experiment{i}";
-                if (!subdirs.Contains(experimentDir))
-                {
-                    break;
-                }
-            }
-            var experimentDirFullPath = Path.Combine(rootDir.FullName, experimentDir);
+
+            var experimentDirFullPath = Path.Combine(rootDir.FullName, Path.GetRandomFileName());
```

Comment:
Can you prefix w/ "experiment_"? The will help users know its purpose.

```suggestion
            var experimentDirFullPath = Path.Combine(rootDir.FullName, "experiment_" + Path.GetRandomFileName());
```

This creates a path of `Temp\Microsoft.ML.AutoML\experiment_w143kxnu.idj\...`.

Reply:
Thanks @justinormont! Added the prefix.